Multisig/UI/Transaction/TransactionDetailsViewController/AdvancedTransactionDetailsViewController.swift

```diff
@@ -28,30 +29,146 @@ class AdvancedTransactionDetailsViewController: UITableViewController {
 
         tableView.registerCell(DetailAccountCell.self)
         tableView.registerCell(DetailExpandableTextCell.self)
+        tableView.registerHeaderFooterView(BasicHeaderView.self)
 
         tableView.rowHeight = UITableView.automaticDimension
         tableView.estimatedRowHeight = 60
+        tableView.estimatedSectionHeaderHeight = BasicHeaderView.headerHeight
+        tableView.estimatedSectionFooterHeight = 0
+    }
+
+    func buildSections(_ tx: SCGModels.TransactionDetails) {
+        if let hash = tx.txHash?.description {
+            sections.append(Section(title: "Chain transaction data",
+                                    items: [SectionItem(title: "Transaction hash:", value: hash)]))
+        }
+
+        if let txData = tx.txData {
+            var safeTransactionData:[SectionItem] = []
+            safeTransactionData.append(SectionItem(title: "To:", value: txData.to))
+            safeTransactionData.append(SectionItem(title: "Value:", value: txData.value.description))
+            if let data = txData.hexData {
+                safeTransactionData.append(SectionItem(title: "Data:", value: data))
+            }
+
+            // TODO: Check
+//            if let data = txData.dataDecoded {
+//                safeTransactionData.append(SectionItem(title: "Decoded Data:", value: data))
+//            }
+
+            safeTransactionData.append(SectionItem(title: "Operation:", value: txData.operation.name))
+
+            sections.append(Section(title: "Safe transaction data", items: safeTransactionData))
+        }
+
+        let nonce: String?
+        let safeTxHash: String?
+
+        if case SCGModels.TransactionDetails.DetailedExecutionInfo.multisig(let multisigTx)? =
+            tx.detailedExecutionInfo {
+            nonce = multisigTx.nonce.description
+            safeTxHash = multisigTx.safeTxHash.description
+        } else {
+            nonce = nil
+            safeTxHash = nil
+        }
+
+        if let nonce = nonce, let safeTxHash = safeTxHash {
+            sections.append(Section(title: "",
+                                    items: [SectionItem(title: "safeTxHash:", value: safeTxHash),
+                                            SectionItem(title: "Nonce:", value: nonce)]))
+        }
```

Comment:
these constants are effectively used only for the case when they exist, so I'd refactor it this way:

```suggestion
        if case SCGModels.TransactionDetails.DetailedExecutionInfo.multisig(let multisigTx)? = tx.detailedExecutionInfo {
            sections.append(Section(title: "",
                                    items: [SectionItem(title: "safeTxHash:", value: multisigTx.safeTxHash.description),
                                            SectionItem(title: "Nonce:", value: multisigTx.nonce.description)]))
        }
```

Reply:
This way the section will be always there even if there is no value for both, I have refactored it in different way